packages/tracing/src/browser/metrics.ts

```diff
@@ -214,7 +214,8 @@ export class MetricsInstrumentation {
       transaction.setTag('lcp.size', this._lcpEntry.size);
     }
 
-    if (this._clsEntry) {
+    // See: https://developer.mozilla.org/en-US/docs/Web/API/LayoutShift
+    if (this._clsEntry && this._clsEntry.sources) {
```

Comment:
```suggestion
    if (this._clsEntry?.sources) {
```

The tiniest of nits. I just really like optional chaining. 😛

Reply:
Not sure it's worth the extra code generated from typescript: https://www.typescriptlang.org/play?ts=3.7.5#code/MYewdgzgLgBFAWAuGBvGB9YAbCBRMUATgJ4D8yaEIArocAKYTkzSECWYA5gNoC6MAX0EwAvKgEBYAFDSA9LJgB5AA5Q24AIZYYAYXgaOHTtLYAzGAAoEAOkw58RMtaq0GEAJTjpchQCV6nNRYGoQwoGAAJmxq4CbmVvC22HgEJDAAZOlwiXYpjs40dIyeKJIyUkA, especially if every byte counts. 

Thought that is why we shied away from optional chaining outside of tests?

Reply:
We kinda started to use it out of convenience tbh. It's a shame that only ES2020 benefits from its smaller footprint, because we don't target it yet. I don't like it, but we might want to verify what's the impact of it and if it's unreasonable, us eslint to prevent usage :(

Reply:
![image](https://user-images.githubusercontent.com/1523305/124092911-a3cc0980-da57-11eb-8c36-49a092a8f814.png) (with/without chaining)